report_thesis/src/sections/background.tex

```diff
@@ -15,19 +15,16 @@ \subsubsection{Max Absolute Scaler}
 This maintains the sparsity of the data by not introducing non-zero values in the zero entries of the data~\cite{Vasques2024}.
 
 
-
-\subsubsection{Min-Max Normalization}
-Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$. 
+\subsubsection{Min-Max Normalization}\label{subsec:min-max}
+Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$, where $a$ and $b$ represent the new minimum and maximum values, respectively. 
 The goal is to normalize the range of the data to a specific scale, typically 0 to 1. 
 Mathematically, min-max normalization is defined as:
-
 $$ 
-	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (new max - new min) + new min,
+	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (b - a) + a,
 $$
+where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively. 
 
-where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively, and $new min$ and $new max$ define the range of the transformed data. 
-This type of normalization is particularly useful when algorithms, such as neural networks, are sensitive to the magnitude of input features. 
-Normalizing ensures that each feature contributes equally to the result, leading to more effective and faster convergence.
+This type of normalization is beneficial because it ensures that each feature contributes equally to the analysis, regardless of its original scale. This balanced contribution helps improve the performance and convergence speed of many algorithms, making the learning process more effective and efficient.
```

Comment:
```suggestion
This type of normalization is beneficial because it ensures that each feature contributes equally to the analysis, regardless of its original scale.
```